src/createAnimatedComponent.js

```diff
@@ -15,6 +17,26 @@ function listener(data) {
   component && component._updateFromNative(data.props);
 }
 
+function getEventNode(node) {
+  if (node instanceof AnimatedEvent) {
+    return node;
+  } else if (node instanceof AnimatedCallFunc) {
+    throw new Error('events nested in procs are not yet supported');
```

Comment:
So we throw an error and then do sth after the error?

Reply:
for future usage.
I'm thinking about refactoring `event` to completely extends `map`

Reply:
I'd rather drop it as well. It's not good to do anything after throwing an error

Reply:
removed

Reply:
8080646